style/ember/README.md

```diff
@@ -3,6 +3,11 @@ Ember
 
 * Don't put a space between the opening handlebars braces and the variable.
 * Prefer components over partials.
+* Prefer `ember-fetch` over `ic-ajax`. ([sample][ember-fetch-sample])
+  [documentation][ember-fetch]
```

Comment:
I'm not sure if this is valuable to have this link or not? It'd be real easy to google if someone wanted to find out more.